scripts/linkall.js

```diff
@@ -11,7 +11,7 @@ var packagesdir = path.join(__dirname, '../packages');
 var examplesdir = path.join(__dirname, '../examples');
 var rootdir = path.join(__dirname, '..');
 
-npm(['install', 'async'], rootdir, function() {
+npm(['install'], rootdir, function() {
```

Comment:
This also helps run test with a clean close of the repo. Async is now on devDependencies, so no need to force install separately.